sdl_android/src/main/java/com/smartdevicelink/api/screen/ScreenManager.java

```diff
@@ -0,0 +1,343 @@
+package com.smartdevicelink.api.screen;
+
+import android.support.annotation.NonNull;
+import android.util.Log;
+
+import com.smartdevicelink.api.BaseSubManager;
+import com.smartdevicelink.api.CompletionListener;
+import com.smartdevicelink.api.FileManager;
+import com.smartdevicelink.api.datatypes.SdlArtwork;
+import com.smartdevicelink.proxy.interfaces.ISdl;
+import com.smartdevicelink.proxy.rpc.SetDisplayLayout;
+import com.smartdevicelink.proxy.rpc.enums.MetadataType;
+import com.smartdevicelink.proxy.rpc.enums.PredefinedLayout;
+import com.smartdevicelink.proxy.rpc.enums.TextAlignment;
+
+import java.util.List;
+
+/**
+ * <strong>ScreenManager</strong> <br>
+ *
+ * Note: This class must be accessed through the SdlManager. Do not instantiate it by itself. <br>
+*/
+public class ScreenManager extends BaseSubManager {
+
+	private static String TAG = "ScreenManager";
+	private FileManager fileManager;
```

Comment:
This should be a `WeakReference` since the original lives in the `SdlManager` and the screen manager shouldn't keep the `FileManager` from being GC'ed.

Reply:
The reference can also be `final`